python/ommx/src/constraint.rs

```diff
@@ -0,0 +1,213 @@
+use crate::message::Function;
+use anyhow::Result;
+use fnv::FnvHashMap;
+use ommx::{ConstraintID, Equality, Message, Parse};
+use pyo3::{prelude::*, types::PyBytes};
+use std::collections::HashMap;
+
+/// Constraint wrapper for Python
+#[cfg_attr(feature = "stub_gen", pyo3_stub_gen::derive::gen_stub_pyclass)]
+#[pyclass]
+#[derive(Clone)]
+pub struct Constraint(pub ommx::Constraint);
+
+#[cfg_attr(feature = "stub_gen", pyo3_stub_gen::derive::gen_stub_pymethods)]
+#[pymethods]
+impl Constraint {
+    #[new]
+    #[pyo3(signature = (id, function, equality, name=None, subscripts=None, description=None, parameters=HashMap::default()))]
+    pub fn new(
+        id: u64,
+        function: Function,
+        equality: u32,
+        name: Option<String>,
+        subscripts: Option<Vec<i64>>,
```

Comment:
```suggestion
        subscripts: Vec<i64>,
```

デフォルト値は `Vec::new`にしましょう